components/clp-package-utils/clp_package_utils/scripts/native/compress.py

```diff
@@ -136,31 +137,34 @@ def _generate_clp_io_config(
     input_type = clp_config.logs_input.type
 
     if InputType.FS == input_type:
+        if len(logs_to_compress) == 0:
+            raise ValueError(f"No input logs found.")
```

Comment:
```suggestion
            raise ValueError(f"No input paths given.")
```
Switched to "given" since I feel like "found" implies that the files just don't exist on disk.